src/workflows-app/utils/anonymous-username-pairs.js

```diff
@@ -0,0 +1,186 @@
+export const leftNames = [
+  'admiring',
+  'adoring',
+  'affectionate',
+  'agitated',
+  'amazing',
+  'angry',
+  'awesome',
+  'beautiful',
+  'blissful',
+  'bold',
+  'boring',
+  'brave',
+  'busy',
+  'charming',
+  'clever',
+  // 'compassionate',
```

Comment:
Are these commented out because they are considered "tricky"?

Reply:
they're commented out for various reasons: some because they're a little too long for the column size I picked (and I didn't want to make it wider) and some because of connotations that I thought would be better to avoid

I left them commented so that readers would know that it was there, and explicitly omitted